### DISCUSSION TITLE
Tianji LXC not recheable after a reboot

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#3771] [Help Needed]
Hi

I use Tianji LXC; and i discovered than Tianji LXC is not recheable after a LXC stop or a proxmox reboot
Is there a commad line to start ?

**Comment by fmedinaq** [comment]
Same issue here

**Reply by Narkowii** [reply]
i try a   `pm2 update` in the LXC console; and it works again

**Reply by MickLesk** [reply]
pm2 start /opt/tianji/src/server/dist/src/server/main.js --name tianji
pm2 save

**Comment by Hawolex** [comment]
Have the same issue. ran the command but the servoce dont come back up. not on domain or the LXC ip:12345.
Any ideas to where u start looking now?

![Screenshot 2024-10-02 at 20 20 11](https://github.com/user-attachments/assets/b0fec0dd-95e4-4769-a584-95da42c80c80)

**Comment by tteck** [comment]
**Tianji LXC** [(Commit)](https://github.com/tteck/Proxmox/commit/4c83a790ac9b040da1f11ad2cbe13d3fc5f480e9)
- Breaking Change
- Switch from `pm2` process management to `systemd`
